flight/Modules/Actuator/actuator.c

```diff
@@ -704,6 +715,18 @@ static void normalize_input_data(uint32_t this_systime,
 
 	*stabilize_now = throttle_val != 0.0f;
 
+	if (*flip_over_mode) {
+		apply_channel_deadband(&desired.Pitch, 0.25f);
+		apply_channel_deadband(&desired.Roll, 0.25f);
+		apply_channel_deadband(&desired.Yaw, 0.25f);
```

Comment:
So any input is gonna be allowed? Not just the most prominent axis?

Reply:
Yes, any input.  This is important, because e.g. +'s want to flip over diagonally.  BF has some weird code to detect diagonal situations.

The actual flight mode disables yaw, though.